language-support/scala/bindings-akka-testing/src/main/scala/com/digitalasset/ledger/client/testing/AkkaTest.scala

```diff
@@ -35,8 +35,8 @@ trait AkkaTest extends BeforeAndAfterAll with LazyLogging { self: Suite =>
   protected implicit val scheduler: Scheduler = system.scheduler
   protected implicit val schedulerService: ScheduledExecutorService =
     Executors.newSingleThreadScheduledExecutor()
-  protected implicit val materializer: ActorMaterializer = ActorMaterializer(
-    ActorMaterializerSettings(system).withSupervisionStrategy(decider(system)))
+  protected implicit val materializer: Materializer = Materializer(system)
+  ActorAttributes.supervisionStrategy(decider(system))
```

Comment:
Looks like [`ActorAttributes.supervisionStrategy`, etc. are pure functions](https://github.com/akka/akka/blob/v2.6.1/akka-stream/src/main/scala/akka/stream/Attributes.scala#555), so this doesn't do anything any more. If everything is still working with this change, we can probably delete it (and `decider`) entirely.

I don't know why this was done in the first place, though, so I'd be wary. If we want to make sure this still works, we need to set it on the stream itself:

```scala
Source(…)
  .blah
  .toMat(sink)(Keep.right)
  .withAttributes(ActorAttributes.supervisionStrategy(decider(system))
```

Reply:
removed